src/focus.service.ts

```diff
@@ -368,14 +368,22 @@ export class FocusService {
     });
 
     if (isNodeAttached(this.selected, this.root)) {
-      this.bubbleEvent(ev);
+      this.bubbleEvent(ev, false);
     }
 
     // Abort if the user handled
     if (ev.defaultPrevented) {
       return true;
     }
 
+    // Bubble once more on the target.
+    if (ev.next) {
+      this.bubbleEvent(ev, true, ev.next);
```

Comment:
This is handled in [bubbleEvent's loop](https://github.com/WatchBeam/arcade-machine/blob/9f0f3ba4bef07939d3977f11451313a415eeb249/src/focus.service.ts#L492)